gatsby-browser.js

```diff
@@ -54,5 +54,16 @@ const onClientEntry = () => {
   window.React = require('react');
   window.ReactDOM = require('react-dom');
 };
+const onRouteUpdate = ({ location, prevLocation }) => {
```

Comment:
I was able to confirm with Andy that the UTM links will be directly to the quickstart itself, instead of to the I/O marketplace. Assuming this code was added to deal with that situation, I think we can remove it. If not, please ignore this comment!